ServiceApp/src/org/droidplanner/core/MAVLink/MavLinkParameters.java

```diff
@@ -25,6 +25,16 @@ public static void sendParameter(Drone drone, Parameter parameter) {
 		drone.getMavClient().sendMavMessage(msg, null);
 	}
 
+	public static void sendParameter(Drone drone, String name, int type, float value) {
```

Comment:
@squilter Update the `sendParameter(Drone drone, Parameter parameter)` so that its body delegates the call to the newly defined `sendParameter(...)`.